Main.sublime-menu

```diff
@@ -208,14 +208,10 @@
         ]
     },
     {
-        "caption": "Preferences",
```

Comment:
As your other pull patched the mnemonics already (which I merged), the only other thing this pull offers is moving the tools entries which I don't really want to do.  I am probably going to pass on this.